packages/angular/.storybook/carbon-charts-theme.ts

```diff
@@ -0,0 +1,8 @@
+import { create } from '@storybook/theming'
+
+export default create({
+	base: 'dark',
+	brandTitle: 'Carbon Charts - Angular',
+	brandTarget: '_self',
+	// fontBase: '"IBM Plex Sans", Arial, sans-serif' // throws off Storybook's icon alignment (only applies to left navigation)
```

Comment:
Only affects the left navigation in storybook - not the stories themselves. While it can be enabled, the navigation icons aren't going to align without some smithing in CSS.

Reply:
Are you ok with leaving the left navigation as it is now?

Reply:
Yeah that's fine. I'll just look into it separately